docs/ethics/review.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: layouts/ethics.njk
+subsection: Ethics
+title: Doing an ethics review
+---
+
+Doing an ethics review will help you:
+
+- design, develop and maintain responsible services
+- identify any unintended consequences of your service
+
+## How to do an ethics review
+
+Make a copy of the ethics review template using [Google Docs](https://docs.google.com/document/d/1GRHSKxj3_zXubfDkGHQp9Y2_6wpf9_dcB9qCuHQS4xM/edit) or [Miro](https://miro.com/app/board/uXjVPlD7OmE=/?userEmail=laura.parker6@digital.justice.gov.uk&track=true&utm_source=notification&utm_medium=email&utm_campaign=add-to-board&utm_content=go-to-board) (both come with instructions).
```

Comment:
Is this URL supposed to have Laura's email address in it?

Reply:
No, I'll get that updated. Great spot, thank you.